### PR TITLE
fix: parse zeroxv3 trade

### PR DESCRIPTION
**Pull request by vaigay**

### CODE REVIEW

**Review by secmask — APPROVED** [review]

File: pkg/parser/zxrfqv3/parser.go [hunk 1]
```diff
@@ -363,3 +373,58 @@ func (p *Parser) LogFromExchange(log ethereumTypes.Log) bool {
...
+		offset += paddingByteSize * 2
+	}
+	if len(data) < offset {
+		return nil, false, ErrorInputDataIsNotEnough
+	}
+	actionsSize := new(big.Int).SetBytes(data[offsetActions : offsetActions+paddingByteSize]).Uint64()
+	baseOffset := int(offsetActions + paddingByteSize)
+	// skip actions size
+	offset += paddingByteSize
+	var eachActionOffset []int
```
Comment (ngocthanh1389) [inline pkg/parser/zxrfqv3/parser.go:412]: init size

File: pkg/parser/zxrfqv3/parser.go [hunk 2]
```diff
@@ -363,3 +373,58 @@ func (p *Parser) LogFromExchange(log ethereumTypes.Log) bool {
...
+
+	for i := 0; i < int(actionsSize); i++ {
+		actionOffset := new(big.Int).SetBytes(data[offset : offset+paddingByteSize]).Uint64()
+		eachActionOffset = append(eachActionOffset, int(actionOffset))
+		offset += paddingByteSize
+	}
+	var actions [][]byte
```
Comment (ngocthanh1389) [inline pkg/parser/zxrfqv3/parser.go:419]: init size

**Review by ngocthanh1389 — APPROVED** [review]

File: pkg/parser/zxrfqv3/parser.go (hunk 2 above)
Comment (vaigay) [reply pkg/parser/zxrfqv3/parser.go]: updated in this commit [9735fdf](https://github.com/KyberNetwork/tradelogs/pull/87/commits/9735fdf06fbf4ffc1f4dfb32eb5c3d73cbfba61f)

File: pkg/parser/zxrfqv3/parser.go (hunk 1 above)
Comment (vaigay) [reply pkg/parser/zxrfqv3/parser.go]: updated in this commit [9735fdf](https://github.com/KyberNetwork/tradelogs/pull/87/commits/9735fdf06fbf4ffc1f4dfb32eb5c3d73cbfba61f)

**Pull request merged**